templates/dashboard/collection/list.html

```diff
@@ -0,0 +1,70 @@
+{% extends 'dashboard/base.html' %}
```

Comment:
Currently, all of our list views in the dashboard provide filtering capabilities, we should add it in this view as well.

Reply:
@elwoodxblues I added option to filter by name, is it enough?

Reply:
Yes, for now, we can filter collections only by name.